### ISSUE TITLE
Allow setting CONAN_USER_HOME for the run commands

### ISSUE THREAD

**Issue by Lawrencemm** [conan-io/conan-clion-plugin#22]
Since we have the option to install configs with this plugin, and I don't want to pollute my global config with a bunch of installed configs from different projects (especially since conan config install actually composes some options by default).

The specified directory should be created if it doesn't exist for convenience.

**Comment by jgsogo** [comment]
I agree that this could be a useful feature (and it is easy to implement), but I would like to know why it is needed. If I am using Conan I would expect to use the same executable, cache, and configuration for all the projects and for the command line. I'm afraid that needing a different Conan configuration per project is a symptom of something else.

What are the options/configs that disturb you with a single global configuration?

Thanks for your feedback

**Comment by Lawrencemm** [comment]
To be honest it's probably just because I wanted to avoid having to specify
a profile all the time when I install, so I set the default profile to
whatever I'm using most in each project.

A bit of a hack so I guess you are right. Having #24 would probably
eliminate the need for this use case.

**Comment by jgsogo** [comment]
Right now I'm using the plugin more often and I feel like this is something useful. If we let the user choose the Conan executable, it will probably make sense to let them use the same executable but different `CONAN_USER_HOME` depending on the projects: open-source projects will use one configuration, projects at a company a different one,... so in my opinion this is something to add, and probably it makes sense to do it now that we will add envvars from https://github.com/conan-io/conan/pull/5989 (Conan v1.21)
